man/fi_info.1.md

```diff
@@ -92,28 +92,184 @@ the fi_info structure, see fi_getinfo(3).
+        comp_order: [ FI_ORDER_NONE ]
+        inject_size: 3840
+        size: 1024
+        iov_limit: 4
+        rma_iov_limit: 4
+        tclass: 0x0
+    fi_rx_attr:
+        caps: [ FI_MSG, FI_RMA, FI_TAGGED, FI_ATOMIC, FI_RECV, FI_REMOTE_READ, FI_REMOTE_WRITE, FI_MULTI_RECV, FI_RMA_EVENT, FI_SOURCE, FI_DIRECTED_RECV ]
+        mode: [  ]
+        op_flags: [  ]
+        msg_order: [ FI_ORDER_RAR, FI_ORDER_RAW, FI_ORDER_RAS, FI_ORDER_WAW, FI_ORDER_WAS, FI_ORDER_SAW, FI_ORDER_SAS, FI_ORDER_RMA_RAR, FI_ORDER_RMA_RAW, FI_ORDER_RMA_WAW, FI_ORDER_ATOMIC_RAR, FI_ORDER_ATOMIC_RAW, FI_ORDER_ATOMIC_WAR, FI_ORDER_ATOMIC_WAW ]
+        comp_order: [ FI_ORDER_NONE ]
+        total_buffered_recv: 0
+        size: 1024
+        iov_limit: 4
+    fi_ep_attr:
+        type: FI_EP_RDM
+        protocol: FI_PROTO_RXD
+        protocol_version: 1
+        max_msg_size: 18446744073709551615
+        msg_prefix_size: 0
+        max_order_raw_size: 18446744073709551615
+        max_order_war_size: 0
+        max_order_waw_size: 18446744073709551615
+        mem_tag_format: 0xaaaaaaaaaaaaaaaa
+        tx_ctx_cnt: 1
+        rx_ctx_cnt: 1
+        auth_key_size: 0
+    fi_domain_attr:
+        domain: 0x0
+        name: mlx5_0-dgram
+        threading: FI_THREAD_SAFE
+        control_progress: FI_PROGRESS_MANUAL
+        data_progress: FI_PROGRESS_MANUAL
+        resource_mgmt: FI_RM_ENABLED
+        av_type: FI_AV_UNSPEC
+        mr_mode: [  ]
+        mr_key_size: 8
+        cq_data_size: 8
+        cq_cnt: 128
+        ep_cnt: 128
+        tx_ctx_cnt: 1
+        rx_ctx_cnt: 1
+        max_ep_tx_ctx: 1
+        max_ep_rx_ctx: 1
+        max_ep_stx_ctx: 0
+        max_ep_srx_ctx: 0
+        cntr_cnt: 0
+        mr_iov_limit: 1
+        caps: [  ]
+        mode: [  ]
+        auth_key_size: 0
+        max_err_data: 0
+        mr_cnt: 0
+        tclass: 0x0
+    fi_fabric_attr:
+        name: IB-0xfe80000000000000
+        prov_name: verbs;ofi_rxd
+        prov_version: 116.0
+        api_version: 1.16
+    nic:
+        fi_device_attr:
+            name: mlx5_0
+            device_id: 0x101b
+            device_version: 0
+            vendor_id: 0x02c9
+            driver: (null)
+            firmware: 20.33.1048
+        fi_bus_attr:
+            bus_type: FI_BUS_UNKNOWN
+        fi_link_attr:
+            address: (null)
+            mtu: 4096
+            speed: 0
+            state: FI_LINK_UP
+            network_type: InfiniBand
+```
+
+To see libfabric related environment variables `-e` option.
+
+```
+# FI_LOG_INTERVAL: Integer
+# Delay in ms between rate limited log messages (default 2000)
+
+# FI_LOG_LEVEL: String
+# Specify logging level: warn, trace, info, debug (default: warn)
+
+# FI_LOG_PROV: String
+# Specify specific provider to log (default: all)
+
+# FI_PROVIDER: String
+# Only use specified provider (default: all available)
```

Comment:
I'd stop this example here.  Consider adding a separate example using the -g option.  For example "fi_info -g tcp" to get a list of environment variables related to tcp.

Reply:
Done.